src/pages/login/login.test.tsx

```diff
@@ -39,6 +40,29 @@ test('triggers login on form submit', () => {
   });
 });
 
+test('triggers login error on form submit', () => {
+  const mockAxiosResponse: AxiosResponse = {
+    data: {
+      non_field_errors: 'Unable to log in with provided credentials.',
+    },
+    status: 1,
+    statusText: '',
+    headers: {},
+    config: {},
+  };
+  const mockAxiosError: AxiosError = {
+    config: {},
+    message: '',
+    name: '',
+    response: mockAxiosResponse,
+  };
+  const view = shallow(<Login {...props} error={mockAxiosError} />);
+  const preventDefault = jest.fn();
+  view.find('form').simulate('submit', { preventDefault });
```

Comment:
This is actually not needed here. The error will be there regardless since you passed it in as props. The login call is covered in another test.